packages/lib-classifier/src/components/Classifier/components/TaskArea/components/Tasks/Tasks.js

```diff
@@ -51,8 +51,11 @@ class Tasks extends React.Component {
       return (
         <Box as='form' gap='small' justify='between' fill>
           {tasks.map((task) => {
-            addAnnotation(task)
-            const annotation = classification.annotation(task)
+            // classifications.addAnnotation(task, value) retrieves any existing task annotation from the store
+            // or creates a new one if one doesn't exist.
+            // The name is a bit confusing.
+            const annotation = addAnnotation(task)
+            task.setAnnotation(annotation)
```

Comment:
Note to self: remember, what's essentially happening here is that the _"initialise default Annotations for this Task"_ (or _"initialise a default instance of the Annotation model for this instance of a Task model"_ ) action has been moved out of the Task _model_ itself and into the Task _component._

This is fine in theory, but keep an eye out for that tricksy React render() and pay attention to when it's called.

Reply:
`task.defaultAnnotation` still exists, I think. The thing to remember is where that annotation is stored. Originally, all annotations were stored in `classification.annotations` but that's not true now that we have subtasks. A Task component might be passed an annotation from `mark.annotations` instead.